ui/app/AppLayouts/Wallet/controls/SwapExchangeButton.qml

```diff
@@ -15,6 +15,8 @@ StatusButton {
     isRoundIcon: true
     radius: height/2
     normalColor: Theme.palette.indirectColor3
+    disabledColor: normalColor
+    opacity: enabled ? 1 : 0.4
```

Comment:
To achieve the disabled effect, we have to work with `opacity` here since our prescribed `normalColor` is (partially) transparent already, thus making the `disabledColor` ineffective

Reply:
sounds reasonable to add a comment about this in code in case someone comes looking in the future